<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxPaint::ColourSpacePlaneWidget</class>
 <widget class="QWidget" name="GfxPaint::ColourSpacePlaneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="colourSpaceLabel">
     <property name="text">
      <string>&amp;Colour Space</string>
     </property>
     <property name="buddy">
      <cstring>colourSpaceComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="colourSpaceComboBox">
     <item>
      <property name="text">
       <string>RGB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sRGB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XYZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>xyY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HSV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HSL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HCY</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QVBoxLayout" name="colourSpacePlaneLayout"/>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="quantiseCheckBox">
     <property name="text">
      <string>Quantise</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="alphaCheckBox">
     <property name="text">
      <string>Alpha</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="xComponentLabel">
     <property name="text">
      <string>X Component</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="xComponentComboBox"/>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="yComponentComboBox"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="yComponentLabel">
     <property name="text">
      <string>Y Component</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
